MapboxDirections/MBRouteOptions.swift

```diff
@@ -304,6 +312,13 @@ open class RouteOptions: NSObject, NSSecureCoding {
     open var includesExitRoundaboutManeuver = false
     
     /**
+     An Array of `RoadClass` that the routing engine will attempt to avoid.
+     
+     The order of this Array does not impact the results.
+     */
+    open var excludeRoadClasses: RoadClasses? = []
```

Comment:
An optional `NS_OPTION` doesn’t bridge to Objective-C. To make this property bridge, we’d have to introduce a `none` RoadClass and make both this property and `Intersection.outletRoadClasses` non-optional.